ros2_batch_job/__main__.py

```diff
@@ -48,7 +48,8 @@
     'pep8',
     'pydocstyle',
     'pyflakes',
-    'flake8',
+    'flake8==2.6.2',
```

Comment:
I would propose to still use the latest release. If they really change the API we will notice immediately and can update our usage.